yarn.lock

```diff
@@ -4241,7 +4241,7 @@
     terminal-link "^2.0.0"
     v8-to-istanbul "^4.1.3"
   optionalDependencies:
-    node-notifier "^9.0.0"
+    node-notifier "^6.0.0"
```

Comment:
Why are we rolling to an older version of node-notifier?  Is this necessary?

Reply:
do you think we should keep the old version? it has a moderate severity. but keep the old version for now.

Reply:
If this mitigates a security warning on a dependency then using 6 is great.  Thanks for the clarification.